File: python_sample/payloads/merge_fields.py
import json

type_fields = {"label": str, "value": str, "font_size": int}


class MergeFields:
    label = ''
    value = ''
    font_size = 0

    def __init__(self, *args, **kwargs):
        mandatory_fields = ["label", "value"]
        for key, val in kwargs.items():
            setattr(self, key, val)

            if key in mandatory_fields:
                mandatory_fields.remove(key)

        if mandatory_fields:
            raise Exception("{} are missing".format(','.join(mandatory_fields)))

    def __str__(self):
        return json.dumps({"label": self.label, "value": self.value, "font_size": getattr(self, "font_size", None)})

    def to_dict(self):
        return {"label": self.label, "value": self.value, "font_size": getattr(self, "font_size", None)}

    def __setattr__(self, name, value):
        if not isinstance(value, type_fields.get(name)):
            raise Exception("{} should be a {}".format(name, str(type_fields.get(name))))
        super().__setattr__(name, value)

File: v2/java_sample/payloads/MergeFields.java
package payloads;

import org.json.JSONObject;

public class MergeFields {
	private String label;
	private String value;
	private int font_size = 0;

	public MergeFields(String label, String value, int font_size)
	{
		this.label = label;
		this.value = value;
		this.font_size  = font_size;
	}
	
	public MergeFields(String label, String value)
	{
		this.label = label;
		this.value = value;
	}
	
	
	public JSONObject toJson() 
	{
		JSONObject obj = new JSONObject();

		obj.put("label", this.label);
		obj.put("value", this.value);
		obj.put("font_size", this.font_size);
		return obj;
		
	}

}
